docs/sources/setup/token.md

```diff
@@ -15,37 +15,35 @@ labels:
 weight: 102
 ---
 
-# Create a GitHub personal access token
+# Create a GitHub access token
 
-You will need a _personal access token_ to use the plugin. GitHub currently supports two types of personal access tokens:
-
-1. fine-grained personal access tokens
-1. personal access tokens (classic)
-
-Read more about [personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
-
-The Grafana GitHub data source plugin works with both. Below is a table that indicates what minimum requirements must be matched before the plugin can be used.
-
-Options:
-
-| Setting               | Required | Description                                           |
-| --------------------- | -------- | ----------------------------------------------------- |
-| Access token          | true     | This is required to allow plugin to connect to GitHub |
-| GitHub Enterprise URL | false    | Only if you are using GitHub Enterprise account       |
+You will need either a `GitHub App` or a `Personal Access Token` to use this plugin.
 
 ## Creating a personal access token (classic)
 
-This is an example when you want to use the personal access token (classic).
+This is an example when you want to use the personal access token (classic). \
+Read more about [personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
 
 1. Login to your GitHub account.
 1. Navigate to [Personal access tokens](https://github.com/settings/tokens) and click **Generate new token**.
 1. Select the **personal access token (classic)**.
 1. Define the permissions which you want to allow.
 1. Click **Generate Token**.
 
-### Permissions
+## Using GitHub App Authentication
+
+You can also authenticate using a GitHub App instead of a personal access token. This method allows for better security and fine-grained access to resources.
+
+1. Register a new GitHub App by following the instructions in the [GitHub App documentation](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app).
+1. After registering the App, generate a private key for authentication.
+1. Note down the App ID assigned to your GitHub App.
+1. Install the GitHub App on your GitHub account or organization.
```

Comment:
Do we want to link https://docs.github.com/en/apps/using-github-apps/installing-your-own-github-app?